bazel/README.md

```diff
@@ -68,19 +68,19 @@ for how to update or override dependencies.
 
     ### Fedora
     On Fedora (maybe also other red hat distros), run the following:
-    ```
+    ```bash
```

Comment:
i think these should be `console` - `bash` is for scripts

```console
$ sudo apt-get install \
       libtool \
       ...
```

rather than

```bash
sudo apt-get install \
       libtool \
       ...
```

Reply:
dont worry about the `$` changing that is out of scope i think as it affects the whole file - but lets use `console`

(docs ref is here https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L5351)